client/java/src/test/java/io/openlineage/client/transports/HttpTransportTest.java

```diff
@@ -209,4 +218,42 @@ void clientClosesNetworkResources() throws IOException {
     verify(response, times(1)).close();
     verify(response.getEntity().getContent(), times(1)).close();
   }
+
+  @Test
+  void customHeaders() throws IOException {
+    HashMap<String, String> headers = new HashMap<>();
+    headers.put(ACCEPT, "not-application/json");
+    headers.put(CONTENT_TYPE, "not-application/json");
+    headers.put("testHeader1", "test1");
+    headers.put("testHeader2", "test2");
+
+    HttpConfig config = new HttpConfig();
+    config.setUrl(URI.create("https://localhost:1500/api/v1/lineage"));
+    config.setHeaders(headers);
+
+    CloseableHttpClient http = mock(CloseableHttpClient.class);
+    Transport transport = new HttpTransport(http, config);
+    OpenLineageClient client = new OpenLineageClient(transport);
+
+    CloseableHttpResponse response = mock(CloseableHttpResponse.class, RETURNS_DEEP_STUBS);
+    when(response.getStatusLine().getStatusCode()).thenReturn(200);
+    when(response.getEntity().isStreaming()).thenReturn(true);
+    Map<String, HttpPost> map = new HashMap<>();
```

Comment:
Isn't it just a map with a singe entry with a key `test`? If so, why do we need it?